action-scripts/build-config.js

```diff
@@ -57,11 +57,7 @@ const buildConfig = async ({
     platforms: { [styleSystem]: platforms[styleSystem] },
   };
 
-  await fs.writeFile("config.json", JSON.stringify(config), (err) => {
-    if (err) {
-      console.error(err);
-    }
-  });
+   fs.writeFileSync("config.json", JSON.stringify(config));
```

Comment:
`build` was running as soon as `config.json` was created, but before any data was written to the file  - causing an error in `build` since `config.json` was an empty file.  `writeFileSync` allows the file to be created, and data written, before `build` is called.